apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -845,7 +845,8 @@ function initializeBlocklyWrapper(blocklyInstance: GoogleBlocklyInstance) {
     blocklyWrapper.disableVariableEditing =
       !!optOptionsExtended.disableVariableEditing;
     blocklyWrapper.varsInGlobals = !!optOptionsExtended.varsInGlobals;
-    blocklyWrapper.isStartMode = !!optOptionsExtended.editBlocks;
+    blocklyWrapper.isStartMode =
+      optOptionsExtended.editBlocks === 'start_sources';
```

Comment:
This was previously using a double `!` to falsely determine we were in start mode with any `editBlocks` value. Determining this correctly is a needed in order to have some settings apply to Start Mode but not Toolbox Mode. Compare to line 850, below.